data/DSLAB/projects.yaml

```diff
@@ -189,9 +234,30 @@ projects:
         tags:
             - Cloud
             - Performance
+        information:
+          - type: Paper
+            title: "Parallel Symbolic Execution for Automated Real-World Software Testing"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
+            notes:
+              - label: Published in
+                text: "EuroSys 2011"
+                url: http://eurosys2011.cs.uni-salzburg.at/pdf/eurosys2011-salomie.pdf
+          - type: Paper
+            title: "Automated Software Testing as a Service"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
+            notes:
+              - label: Published in
+                text: "SoCC 2010"
+          - type: Paper
+            title: "Cloud9: A Software Testing Service"
+            url: http://dslab.epfl.ch/pubs/cloud9-ladis.pdf
+            notes:
+              - label: Published in
+                text: "ACM Operating Systems Review 2009"
+                url: http://www.sigops.org/osr.html
```

Comment:
It's a 404 for me

Reply:
Same here.
It seems available at: https://dl.acm.org/toc/sigops/2010/43/4